Modules/_testcapimodule.c

```diff
@@ -4690,6 +4690,40 @@ get_mapping_items(PyObject* self, PyObject *obj)
     return PyMapping_Items(obj);
 }
 
+static PyObject *
+test_mapping_has_key_string(PyObject *self, PyObject *Py_UNUSED(args))
```

Comment:
Don't we need to call this function from `test_capi.py`? Or is that automatic?

Reply:
It is: https://github.com/python/cpython/blob/964a92ca636fb856c7ba6fee8e0408f3eeaca850/Lib/test/test_capi.py#L1185-L1189